src/tester/upload.py

```diff
@@ -83,9 +81,6 @@ def main():
     )
     storage_client = storage.Client(PROJECT_ID, credentials=credentials)
     bucket = storage_client.bucket("rubin-prompt-proto-main")
-    batch_settings = pubsub_v1.types.BatchSettings(
```

Comment:
We should probably leave this in and add it to the `PublisherClient` constructor below.